tools/devops/automation/templates/tests/stage.yml

```diff
@@ -144,13 +146,18 @@ stages:
           gitHubToken: ${{ parameters.gitHubToken }}
           xqaCertPass: ${{ parameters.xqaCertPass }}
 
-  - job: publish_html
-    displayName: 'Publish Html report in VSDrops'
+  - job: publish_test_results
+    ${{ if eq(parameters.parseLabels, true) }}:
+      condition: and(eq(dependencies.configure.outputs['labels.skip-all-tests'], ''),  succeededOrFailed())
+    ${{ else }}:
+      condition: succeededOrFailed()
+    displayName: 'GiHub comment - Publish results'
     timeoutInMinutes: 1000
     dependsOn: # has to wait for the tests to be done AND the data to be uploaded
+    - ${{ if eq(parameters.parseLabels, true) }}:
+      - configure
     - ${{ each label in parameters.simTestsConfigurations }}:
       - tests_${{ replace(label, '-', '_') }}
-    condition: succeededOrFailed()
```

Comment:
Was this removal intentional? It seems we won't get the GH comment in case of failures now.